src/ui/components/common/vote_buttons/comment_vote_buttons.rs

```diff
@@ -0,0 +1,39 @@
+use crate::ui::components::common::vote_buttons::VoteButtons;
+use lemmy_client::{
+  lemmy_api_common::{
+    comment::{CommentResponse, CreateCommentLike},
+    lemmy_db_schema::newtypes::CommentId,
+  },
+  LemmyRequest,
+};
+use leptos::*;
+
+#[server(prefix = "/serverfn")]
+async fn vote_comment(id: CommentId, score: i16) -> Result<CommentResponse, ServerFnError> {
+  use crate::{constants::AUTH_COOKIE, utils::get_client_and_session};
+
+  let (client, session) = get_client_and_session().await?;
+
+  let jwt = session.get::<String>(AUTH_COOKIE)?;
```

Comment:
Might be helpful to add another impl for session, with a `session.get_jwt()` function.